lang/src/org/partiql/lang/eval/builtins/ExtractExprFunction.kt

```diff
@@ -17,19 +17,20 @@ package org.partiql.lang.eval.builtins
 import com.amazon.ion.*
 import org.partiql.lang.eval.*
 import org.partiql.lang.syntax.*
+import java.time.LocalDate
 
 private const val SECONDS_PER_MINUTE = 60
 
 /**
- * Extracts a date part from a timestamp where date part is one of the following keywords:
+ * Extracts a date part from a datetime type where date part is one of the following keywords:
  * `year, month, day, hour, minute, second, timestamp_hour, timestamp_minute`.
- *
+ * Datetime type can be one of DATE, TIME or TIMESTAMP
```

Comment:
Nit: `TIME` doesn't exist yet does it?

Reply:
It will in the future. And by future I mean next PR.